assets/files.wxs

```diff
@@ -188,7 +188,7 @@
         <File Id="fil8E8E3BF1C97D92DFEB4FC97D778F238F" KeyPath="yes" Source="$(env.ProductSourcePath)\System.Runtime.CompilerServices.VisualC.dll" />
       </Component>
       <Component Id="cmpf65fc69858294501b4f61951193b392c" Guid="{09C9310D-56AD-4F7A-90C8-DF0F7E8A7251}">
-        <File Id="filDDE80F6ACEC54B56BA33E587A072DE27" KeyPath="yes" Source="$(env.ProductSourcePath)\sos_$(var.FileArchitecture)_$(var.FileArchitecture)_4.6.26725.06.dll" />
+        <File Id="filDDE80F6ACEC54B56BA33E587A072DE27" KeyPath="yes" Source="$(env.ProductSourcePath)\sos_$(var.FileArchitecture)_$(var.FileArchitecture)_4.6.26814.03.dll" />
```

Comment:
@TravisEz13 does the Id on these has to be re-generated? (to be unique for servicing purposes)

Reply:
either the ID or the Guid of the component need to change.  Can you not make the change manually and use the file that `Start-PSPackage` generates and tells you to use?

Reply:
Fixed.